src/media/js/addon/components/addon.js

```diff
@@ -109,15 +119,112 @@ export class Addon extends React.Component {
               <dd>{this.props.latest_version.version}</dd>
             </di>
           }
+
+          {this.props.description &&
+            <di>
```

Comment:
probably time we just make a small component for this

<DefinitionItem type='Description' data={this.props.description}/>